tools/warnings_count/.gitignore

```diff
@@ -2,3 +2,4 @@
 warnings_setup_new.txt
 warnings_disasm_new.txt
 warnings_build_new.txt
+warnings_temp.txt
```

Comment:
I think it does, this may be a leftover from me Jenkins misbehaving and complaining about that file for some reason (I think this didn't fix it tho).
I can revert this if it is preferred.